### PR TITLE
css & updated url

### PR DESCRIPTION
**Pull request by djian001**
- mainly added css and tried to build out front end.
- Updated URL so that others can load my service.

### CODE REVIEW

File: client/src/app.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Reviews extends React.Component {
 
   componentDidMount() {
     $.ajax({
-      url: '/reviews',
+      url: 'http://localhost:3000/reviews',
```
Comment (kimberlyjk) [inline client/src/app.jsx:16]: I would doulbe check this. i'm not running your proxy and i believe your port for service is 3001

File: client/src/reviewList.jsx [hunk 2]
```diff
@@ -1,47 +1,57 @@
 import React from 'react';
-import {Grid, Image, Divider} from 'semantic-ui-react';
+var moment = require('moment');
```
Comment (kimberlyjk) [inline client/src/reviewList.jsx:2]: are you using this instead of semantic UI? curious to hear the reason for the change

**Review by kimberlyjk — APPROVED** [review]
awesome work! 

I had to change the url port to match your service but things rendered on the page once i did that! 

A few notes: 
-  the review dont seem to resort with the sorting function --- you might still be workign that out. 
- there is a warning about the "key" prop that happens i think when you map rthrough an array from React. you might just wnat to add a key={key} in the div tag for each review as you map through the array.

File: client/src/app.jsx (hunk 1 above)
Comment (djian001) [reply client/src/app.jsx:16]: i made the change to 3001 after i made this PR. nice catch!

File: client/src/reviewList.jsx (hunk 2 above)
Comment (djian001) [reply client/src/reviewList.jsx:2]: No, I'm not using semantic UI at all right now. I'm using moment to format my date though

**Pull request merged**